modules/common/src/main/java/com/sygic/modules/common/poi/manager/PoiDataManager.kt

```diff
@@ -43,14 +43,12 @@ interface PoiDataManager {
             }
 
             val reverseSearchResult = results.first()
-            val resultNames = reverseSearchResult.names
-
             onDataLoaded(
                 PoiData(reverseSearchResult.position,
-                    iso = resultNames.countryIso,
-                    city = resultNames.city,
-                    street = resultNames.street,
-                    houseNumber = resultNames.houseNumber)
+                    iso = reverseSearchResult.names.countryIso,
```

Comment:
to `.names` je interne asi getter co? je to sice dtail, ale toto moc nevylepsi performance

Reply:
jj, ale vrací pouze lokální proměnou, žádná logika za tím getterem není. Myslíš, že to bude mít vliv na performance? :)

Reply:
no urcite bude. nie je to mega, ale v takomto pripade mi to pride ako waste... Kazdy getter nieco stoji (aj ked to len vrati premennu) Jasne nie je to taka cena, ze by sme sa vzdali enkapsulacie a vsetko dali public, ale tu z 1 volania urobis 4. Ta lokalna premenna tomu celkom pomahala v tomto pripade

Reply:
https://stuff.mit.edu/afs/sipb/project/android/docs/training/articles/perf-tips.html#GettersSetters
samozrejme s novymi telefonmi je to cim dalej tym menej katastroficke, ale predsa...